operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/Event.java

```diff
@@ -1,8 +1,21 @@
 package io.javaoperatorsdk.operator.processing.event;
 
+/**
+ * The interface all user-defined events must implement. Also implemented by {@link AbstractEvent}.
```

Comment:
Please do not explicitly list implementations. This is automatically handled by javadoc itself.